tsdapiclient/guide.py

```diff
@@ -0,0 +1,89 @@
+
+from tsdapiclient.tools import HELP_URL
+
+topics = """
+config
+uploads
+downloads
+debugging
+"""
+
+config = """
+To use tacl2, you first need to register with the TSD API:
+
+    tacl2 --register
+
+A registration lasts one year, after which you need to register
+again. If you are a member of different TSD projects, you need to
+register separately for each project.
+
+For an overview of current registrations:
+
+    tacl2 --config-show
+
+To delete all existing config:
+
+    tacl2 --config-delete
+
+"""
+
+uploads = """
+Upload a single file:
+
+    tacl2 --upload myfile.txt
```

Comment:
Perhaps a note should be added that the pnum will need to be prepended to most of these commands?

Reply:
oh that was an oversight, thanks!

Reply:
done